Decred Wallet/Custom Views/FloatingLabelTextInput.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  FloatingTextInput.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 03/09/2019.
```

Comment:
Copyright header should follow this pattern: https://github.com/raedahgroup/dcrios/blob/master/Decred%20Wallet/AppDelegate.swift#L5-L7

Reply:
resolved in e7f7df7